internal/config/fw_base.go

```diff
@@ -28,21 +33,21 @@ func (r *RSCommon) Configure(ctx context.Context, req resource.ConfigureRequest,
 		resp.Diagnostics.AddError(errorConfigureSummary, err.Error())
 		return
 	}
-	r.client = client
+	r.Client = client
 }
 
 // DSCommon is used as an embedded struct for all framework data sources. Implements the following plugin-framework defined functions:
 // - Metadata
 // - Configure
 // client is left empty and populated by the framework when envoking Configure method.
```

Comment:
nit: eventually you should update this to `Client` - but see my comment below

Reply:
thx, changed